app/controllers/provider/admin/account/authentication_providers_controller.rb

```diff
@@ -41,14 +50,15 @@ def update
     if @authentication_provider.update(authentication_provider_params)
       redirect_to provider_admin_account_authentication_provider_path(@authentication_provider), notice: 'SSO integration updated'
     else
-      flash.now[:error] = 'SSO integration could not be updated'
+      flash.now[:error] = t('.error')
       render :edit
     end
   end
 
   def destroy
     authentication_provider.destroy
-    redirect_to provider_admin_account_authentication_providers_path, notice: 'SSO integration deleted'
+    flash[:notice] = t('.success')
+    render json: {}
```

Comment:
I think this redirect is actually needed. I tried this:

- Created a new auth provider
- Enter its edit screen
- Click `Delete`

I shows and empty JSON

Reply:
Indeed, good catch @jlledom